server/src/routes/bugsHistory.js

```diff
@@ -1,36 +1,52 @@
 import express from 'express';
+import _ from 'lodash';
+import google from 'googleapis';
+import { spreadsheetID, clientAccountEmail, clientAccountPrivateKey } from '../config/config';
 
+const sheets = google.sheets('v4');
 const router = express.Router();
 
-router.get('/', (req, res, next) => {
-  res.json({
-    status: 'success',
-    message: '',
-    data: {
-      period: 'Last 5 Days',
-      history: [
-        {
-          label: 'Mon',
-          bugs: 12,
-        },
-        {
-          label: 'Tue',
-          bugs: 8,
-        },
-        {
-          label: 'Wed',
-          bugs: 9,
-        },
-        {
-          label: 'Thu',
-          bugs: 10,
-        },
-        {
-          label: 'Fri',
-          bugs: 7,
+const jwtClient = new google.auth.JWT(
```

Comment:
Please see if we can abstract the connection and `get`, `try-catch` logic. We might use Google Sheets as source for another widget.